docs/how-to/redpanda/multi-node/docker-compose.yaml

```diff
@@ -1,68 +1,70 @@
+---
 version: "3.7"
 name: redpanda-quickstart
 networks:
   redpanda_network:
     driver: bridge
 volumes:
-  redpanda-0: null
+  redpanda-0: nullq
 services:
   redpanda-0:
     command:
       - redpanda
       - start
-      - --kafka-addr internal://0.0.0.0:9092,external://0.0.0.0:19092
+      - "--kafka-addr internal://0.0.0.0:9092,external://0.0.0.0:19092"
       # Address the broker advertises to clients that connect to the Kafka API.
       # Use the internal addresses to connect to the Redpanda brokers'
       # from inside the same Docker network.
       # Use the external addresses to connect to the Redpanda brokers'
       # from outside the Docker network.
-      - --advertise-kafka-addr internal://redpanda-0:9092,external://localhost:19092
```

Comment:
The service is still named "redpanda-0", so updating addr flags here to "redpanda" break some networking (however the containers gracefully start). To reproduce, run `docker-compose up -d` in this dir and try hitting `http://localhost:8080`.

I'd suggest replacing any remaining instances of "redpanda-0" with just "redpanda". I tested this locally and could access the console UI as expected.